DataFormats/TestObjects/src/classes_def.xml

```diff
@@ -5,7 +5,9 @@
  <class name="edmtest::IntProduct" ClassVersion="10">
   <version ClassVersion="10" checksum="4286676158"/>
  </class>
- <class name="std::pair<edmtest::IntProduct, edm::RandomNumberGeneratorState>"/>
```

Comment:
Why was this dictionary removed?

Reply:
Actually, no idea, so I guess "by mistake".